app/views/groupTab/components/MemberList.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+import React from 'react'
+import { Text, Button, Container, Content, List, ListItem } from 'native-base'
+import { connect } from 'react-redux'
+
+export default class TaskList extends React.Component {
+  render() {
```

Comment:
Can we put a fake array of members between 6 & 7 and .map over an array calling list Items?